docs/antrea-ipam.md

```diff
@@ -193,18 +194,26 @@ restore its configurations at exit. Node may lose network connection when `antre
 or OVS daemons are stopped unexpectedly, which can be recovered by rebooting the Node.
 `AntreaIPAM` Pods' traffic will not be routed by local Node's network stack.
 
-All traffic to a local Pod will be sent to the Pod's OVS port directly, after the
-destination MAC is rewritten to the Pod's MAC address. This includes `AntreaIPAM` Pods
-and regular `Subnet per Node` IPAM Pods, even they are not in the same subnets.
-Inter-Node traffic will be sent to the Node network from the source Node, and forwarded
-to the destination Node by the Node network.
+Traffic from `AntreaIPAM` Pods without VLAN, regular `Subnet per Node` IPAM Pods, and K8s
+Nodes are recognized as VLAN 0 (untagged).
+
+All traffic to a local Pod in same VLAN will be sent to the Pod's OVS port directly,
+after the destination MAC is rewritten to the Pod's MAC address. This includes
+`AntreaIPAM` Pods and regular `Subnet per Node` IPAM Pods, even when they are not in the
+same subnet.
+All traffic to a Pod in different VLAN will be sent to the gateway of underlay network.
+Inter-Node traffic will be sent to the underlay network from the source Node, and
+forwarded to the destination Node by the underlay network.
 
 ### Requirements for this Feature
 
 As of now, this feature is supported on Linux Nodes, with IPv4, `system` OVS datapath
 type, and `noEncap`, `noSNAT` traffic mode.
 
-The IPs in the `IPPools` must be in the same "underlay" subnet as the Node IP, because
-inter-Node traffic of AntreaIPAM Pods is forwarded by the Node network. Only a single IP
-pool can be included in the Namespace annotation. In the future, annotation of up to two
-pools for IPv4 and IPv6 respectively will be supported.
+The IPs in the `IPPools` without VLAN must be in the same "underlay" subnet as the Node
+IP, because inter-Node traffic of AntreaIPAM Pods is forwarded by the Node network.
+The IPs in the `IPPools` with VLAN must not overlap with other network subnets, and the
+gateway of underlay network should provide the network connectivity for these VLANs.
```

Comment:
Maybe say "the underlay network router" to be more generic.

Reply:
Fixed.